cms/templates/cms/home_page.html

```diff
@@ -68,180 +74,204 @@
-                          {{ counter }} course
-                        {% plural %}
-                          {{ counter }} courses
-                        {% endblocktrans %}
-                      </div>
-                    {% endif %}
-                  {% endwith %}
-                </div>
-                {% if programpage and programpage.thumbnail_image %}
-                  {% image programpage.thumbnail_image fill-690x530 as thumbnail_image %}
-                  <img src="{{ thumbnail_image.url }}" alt=""
-                       >
-                {% else %}
-                  <img src="{% static 'images/course-thumbnail.png' %}"
-                       alt="" class="program-default-image">
-                {% endif %}
-              {% if programpage %}
-                </a>
-              {% endif %}
-            </div>
-            <div class="program-description">
-              <p class="program-description-text" id="program-{{ program.id }}-description">{{ program.description|default:"No description available for this program." }}</p>
-
-              {% if programpage %}
-                <p class="program-description-link">
-                  <a href="{{ programpage.url }}" class="program-link">
-                    More about {{ program }}
-                  </a>
-                </p>
-              {% endif %}
-            </div>
-          </li>
-        {% endfor %}
-
-        </ul>
-        <br clear="all">
 
-        <div class="more-coming-soon">
-          <p>MIT plans to bring more MicroMasters programs to motivated learners like you. Sign up for our mailing list to hear about new MicroMasters programs.</p>
-          <p>
-            <a data-toggle="modal" data-target="#subscribeAlert" class="sign-up-link">Sign Up for Mailing List</a>
-          </p>
+      <section class="events">
+        <div class="container">
+          <h3>Join us for one of our upcoming free events online</h3>
+          <div class="content-holder">
+            <img src="{% static 'images/events.png' %}" alt="Events">
+            <a class="btn btn-primary text-uppercase px-5 py-2" href="#">View Events</a>
+          </div>
         </div>
-
       </section>
 
-      <section class="row reasons-section">
+      <section class="b2b">
+        <div class="container">
+          <img src="{% static 'images/b2b.png' %}" alt="B2B">
+          <div class="content-holder">
+            <h3>For Enterprises and Institutions</h3>
+            <p>
+              Equip your workforce and organization with new applicable skills through the MITx MicroMasters program!
+             </p> 
+             <p>
+              We work closely with organizations globally that wish to integrate our programs into their own education 
+              and training portfolios. Flexible pricing options available. 
+             </p>
+             <a class="btn btn-primary text-uppercase px-5 py-2" href="#">Inquire Now</a>
```

Comment:
This will be _another_ CMS page. I am trying to convince them to give it the URL `for-enterprises-and-institutions` but I'm sure they will change it. 

Again, it's up to you whether to wrap this in a function or hard code it. Since it's a button, it's going to look like you can click on it even when we don't have a URL... 

 ```suggestion
             <a class="btn btn-primary text-uppercase px-5 py-2" href="for-enterprises-and-institutions/">Inquire Now</a>
```